include/nbl/ui/KeyCodes.h

```diff
@@ -0,0 +1,26 @@
+
+namespace nbl::ui
+{
+	enum E_KEYCODE
+	{
+		// TODO
+	};
+	enum E_KEYMODIFIER
+	{
+
+	};
+	struct SKeyInfo
+	{
+		E_KEYCODE keyCode;
+		E_KEYMODIFIER keyModifier;
+	};
+
+	enum E_MOUSEBUTTON
+	{
+		EMB_LEFT_BUTTON,   // 
+		EMB_RIGHT_BUTTON,  // Or should they all be EMB_BUTTON_NUMBER??
```

Comment:
its ok like it is,
what are BUTTON_4 and BUTTON_5 actually?

Reply:
Just additional mouse buttons. like gaming mice usually have two more buttons on the side, but i'm actually wondering why it is limited to 5 in WinAPI cause i'm almost sure i've seen more.